File: app/src/main/java/com/example/acplite/utilidades/UtilidadesArbol.java
package com.example.acplite.utilidades;

public class UtilidadesArbol {
    public static final String DB_NAME = "ACPLiteDB";
    public static final String TREES_TABLE_NAME = "Arboles";
    public static final String CAMPO_NOMBRE = "nombre";
    public static final String CAMPO_NOMBRE_CIENTIFICO = "nombreCientifico";
    public static final String CAMPO_DESCRIPCION = "descripcion";
    public static final String CAMPO_IMAGEN = "imagen";

    public static final String CREATE_TABLE_TREE = "CREATE TABLE " + TREES_TABLE_NAME + " ( " +
            CAMPO_NOMBRE + " TEXT, " +
            CAMPO_NOMBRE_CIENTIFICO + " TEXT, " +
            CAMPO_DESCRIPCION + " TEXT, " +
            CAMPO_IMAGEN + " BLOB)";


    public static final String UPDATE_TABLE_TREE = "DROP TABLE IF EXISTS " + TREES_TABLE_NAME;
}

File: app/src/main/java/com/example/acplite/utilidades/UtilidadesArbol.kt
package com.example.acplite.utilidades

object UtilidadesArbol {
    const val DB_NAME = "ACPLiteDB"
    const val TREES_TABLE_NAME = "Arboles"
    const val CAMPO_NOMBRE = "nombre"
    const val CAMPO_NOMBRE_CIENTIFICO = "nombreCientifico"
    const val CAMPO_DESCRIPCION = "descripcion"
    const val CAMPO_IMAGEN = "imagen"
    const val CREATE_TABLE_TREE = "CREATE TABLE " + TREES_TABLE_NAME + " ( " +
            CAMPO_NOMBRE + " TEXT, " +
            CAMPO_NOMBRE_CIENTIFICO + " TEXT, " +
            CAMPO_DESCRIPCION + " TEXT, " +
            CAMPO_IMAGEN + " BLOB)"
    const val UPDATE_TABLE_TREE = "DROP TABLE IF EXISTS " + TREES_TABLE_NAME
}